src/components/Calculator.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+
+      <div className="calculator card">
+
+        <input type="text" className="calculator-screen z-depth-1" placeholder="0" disabled />
+
+        <div className="calculator-keys">
+
+          <button type="button" className="operator btn btn-secondary">+/-</button>
+          <button type="button" className="operator btn btn-secondary">AC</button>
+          <button type="button" className="operator btn btn-secondary">%</button>
+          <button type="button" className="operator btn btn-secondary">&divide;</button>
+
+          <button type="button" className="btn btn-gray waves-effect">7</button>
+          <button type="button" className="btn btn-gray waves-effect">8</button>
+          <button type="button" className="btn btn-gray waves-effect">9</button>
+          <button type="button" className="operator btn btn-Orange">+</button>
+
+          <button type="button" className="btn btn-gray waves-effect">4</button>
+          <button type="button" className="btn btn-gray waves-effect">5</button>
+          <button type="button" className="btn btn-gray waves-effect">6</button>
+          <button type="button" className="operator btn btn-Orange">-</button>
+
+          <button type="button" className="btn btn-gray waves-effect">1</button>
+          <button type="button" className="btn btn-gray waves-effect">2</button>
+          <button type="button" className="btn btn-gray waves-effect">3</button>
+          <button type="button" className="operator btn btn-Orange">&times;</button>
+
+          <button type="button" className="btn btn-gray zero">0</button>
+          <button type="button" className="decimal function btn btn-secondary dot">.</button>
+          <button type="button" className="equal-sign operator btn btn-Orange equal">=</button>
+
+        </div>
+      </div>
```

Comment:
- [ ] I think it would be nice to have a matching design with what is in the requirements. Currently, the buttons in the second row have different colors and buttons are missing borders around them as shown below:
#### Your design
![image](https://user-images.githubusercontent.com/61377322/148163931-4b3cd454-6c7e-4df2-87a1-bb7756959305.png)

#### Design given
![image](https://user-images.githubusercontent.com/61377322/148164055-6443f5e3-3956-4328-a251-9ff9cfd23bf5.png)

Reply:
This change has been done well